bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -1259,6 +1261,11 @@ impl Client {
         SyncServiceBuilder::new((*self.inner).clone(), self.utd_hook_manager.get().cloned())
     }
 
+    pub fn spaces_service(&self) -> Arc<SpaceService> {
```

Comment:
Minor nit: This would be more consistent with the types.
```suggestion
    pub fn space_service(&self) -> Arc<SpaceService> {
```